python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt_tests/dlt_tests/test_op.py

```diff
@@ -0,0 +1,39 @@
+import dlt
+import duckdb
+from dagster import OpExecutionContext, job, op
+from dagster_embedded_elt.dlt import DagsterDltResource
+
+from .dlt_test_sources.duckdb_with_transformer import pipeline
+
+EXAMPLE_PIPELINE_DUCKDB = "example_pipeline.duckdb"
```

Comment:
Probably want to make this a fixture and ensure that its memory is cleared before test invocation.

Reply:
Created a separate PR with a refactor to used a shared fixture w/ setup / teardown.

https://github.com/dagster-io/dagster/pull/21199/files

Not entirely sure I like how I'm returning a tuple and have to destructure it, open to suggestions.